application/views/themeOptions/options_core.php

```diff
@@ -121,8 +121,13 @@
                                 $optionsValues = !empty($attribute['options']) ? explode('|', $attribute['options']) : array();
                                 $optionLabels = !empty($attribute['optionlabels']) ? explode('|', $attribute['optionlabels']) : array();
                                 $options = array_combine($optionsValues, $optionLabels);
-                                if ($bInherit && isset($sParentOption)){
-                                    $options['inherit'] = gT("Inherit") . ' [' . $sParentOption . ']';
+                                if ($bInherit && isset($sParentOption)) {
+                                    if(is_numeric($sParentOption) && array_key_exists($sParentOption, $options)) {
+                                        $sParentLabelOption = $options[$sParentOption];
+                                        $options['inherit'] = gT("Inherit") . ' [' . gT($sParentLabelOption) . ']';
+                                    } else {
+                                        $options['inherit'] = gT("Inherit") . ' [' . gT($sParentOption) . ']';
```

Comment:
Bug here if sParentOption is empty, it seems.

Reply:
Tim fixed it in latest release.

Reply:
How was this tested, @gabrieljenik ???

Reply:
I am noticing it. I guess that the tested case was the bug one. Sorry

Reply:
On the else part of the if statement, we were supposed to leave the original statement:
`$options['inherit'] = gT("Inherit") . ' [' . $sParentOption . ']';`